.changeset/itchy-spoons-tan.md

```diff
@@ -0,0 +1,6 @@
+---
+"@blitzjs/rpc": patch
+"blitz": patch
+---
+
+Allow for custom page extensions for the wildcard blitz route. For example [...blitz].api.ts
```

Comment:
Maybe we could add a link to the Next docs: https://nextjs.org/docs/api-reference/next.config.js/custom-page-extensions